bitnami/ghost/templates/secrets.yaml

```diff
@@ -1,19 +1,15 @@
 apiVersion: v1
```

Comment:
If .Values.existingSecret is set this should not be created, right? 

Well, I see an issue here because that would force the user to have both the ghostPassword and smtpPassword in the same secret.

Reply:
> If .Values.existingSecret is set this should not be created, right?
I agree, and it is implementing.

Regarding, Let the user have different secrets per each secret implies to have the same logic for all secret data also in the deployment and to have them separately in values too, right? I don't know if it is a common use case and how we are handling it in other charts.